QML_HUD/main.cpp

```
@@ -1,20 +1,28 @@
 #include "vbar.h"
+#include "hboard.h"
 
 #include <QGuiApplication>
 #include <QQmlApplicationEngine>
 #include <QQmlContext> 
 #include <QObject>
 #include <QDebug>
 
+// TODO: Umiescic w QMLach wyswietlacz wartosci i jednostki
+// TODO: Mirror
+// TODO: Poprawic rysowanie niektorych kontrolerow
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)